Automattic/pocket-casts-android
# Rename lifetime member to Pocket Casts Champion

## Description
- Rename `Lifetime member` to `Pocket Casts Champion`
- Opens a dialog when the user taps on `Pocket Casts Champion`
- See PT: pdeCcb-67m-p2
- See figma: G60UqxPhTynbalo0Vfr1NC-fi-2515_9059

## Testing Instructions
1. Log with a lifetime user account. See PdNHOb-2Q-p2
2. Go to profile and tap on `Pocket Casts Champion`
3. ✅ Ensure you see the dialog
4. Tap on Rate button
5. ✅ Ensure you are redirected to play store

## Screenshots or Screencast 

<img src="https://github.com/user-attachments/assets/b92bddd8-d88b-461a-ba9a-a36ff94f666f" width="400">

![image (1)](https://github.com/user-attachments/assets/a913e1be-e2a6-4a9f-8674-bf3054831222)

## Checklist
 
#### I have tested any UI changes...
